xbmc/pvr/PVRGUIActions.h

```diff
@@ -68,6 +68,13 @@ namespace PVR
     bool ShowEPGInfo(const CFileItemPtr &item) const;
 
     /*!
+     * @brief Open a dialog with the epg list for a given item.
+     * @param item containing the channel. item must a channel.
```

Comment:
"must a " -> "must be a"

EDIT: despite from this, item can be either a channel, an epg tag or a timer. CPVRItem::GetChannel() takes care of the conversion. You are actually feeding this method with an epg tag from CGUIDialogPVRGuideInfo.